crates/polars-time/src/chunkedarray/rolling_window/rolling_kernels/no_nulls.rs

```diff
@@ -32,7 +33,7 @@ where
             result.map(|(start, len)| {
                 let end = start + len;
 
-                if start == end {
+                if len < ((min_periods as u32).into()) {
```

Comment:
currently the default is for `min_period` to be `1` if not specified, so this is equivalent to that

https://github.com/pola-rs/polars/blob/ec8c9e02a9f4cbd0bce107a83a3451d06681aea9/py-polars/polars/expr/expr.py#L9976-L9980

Reply:
Can we add a comment about this rationale?

And what is the `into()` going to?